imports/plugins/core/ui-tagnav/client/containers/tagNavContainer.js

```diff
@@ -259,11 +263,13 @@ const wrapComponent = (Comp) => (
       this.setState({ attachedBodyListener: false });
     }
 
+    // TODO: If this is a method of TagNavContainer, why is it, and many
+    // others, declared as an arrow function?
```

Comment:
I don't know if a TODO code comment is the best place to get a programming question answered.